modules/factoidaddress/factoidaddress.go

```diff
@@ -0,0 +1,51 @@
+package factoidaddress
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/sha256"
+	"fmt"
+)
+
+var FactoidAddressPrefix = [2]byte{0x5f, 0xb1}
```

Comment:
i'd prefer making it a slice, no reason for it to be an array, just makes the code more confusing later on

Reply:
:+1: